Fix failure to compile due to outdated code (this code never runs)

diff --git a/external/VSRTL/external/cereal/include/cereal/external/rapidjson/document.h b/external/VSRTL/external/cereal/include/cereal/external/rapidjson/document.h
index 46faeb1..496bb70 100644
--- a/external/VSRTL/external/cereal/include/cereal/external/rapidjson/document.h
+++ b/external/VSRTL/external/cereal/include/cereal/external/rapidjson/document.h
@@ -316,7 +316,7 @@ struct GenericStringRef {
 
     GenericStringRef(const GenericStringRef& rhs) : s(rhs.s), length(rhs.length) {}
 
-    GenericStringRef& operator=(const GenericStringRef& rhs) { s = rhs.s; length = rhs.length; }
+    GenericStringRef& operator=(const GenericStringRef& rhs) = delete;
 
     //! implicit conversion to plain CharType pointer
     operator const Ch *() const { return s; }
